looker_style_guide.md

```diff
@@ -0,0 +1,184 @@
+# LookML Style Guide
+
+
+## Structure of our LookML project
+When structuring our LookML project, our views' folders should loosely reflect the structure of our dbt project with the name of the views representing our business units. For example, we have our core business units (e.g. customers) and our dbt Cloud business units (e.g. accounts, cloud projects, etc) which are similar to our [marts folder](https://github.com/fishtown-analytics/internal-analytics/tree/main/models/marts).
+
+Think of views as raw ingredients that are in the vegetable/fruit aisles in a grocery store. They are ingredients that can be combined and packaged into a product that fits our customers' needs. These "packaged items" are explores. More on that below.
+
+```
+fishtown project
+├── core
+│   └── customers.view
+├── csm
+│   └── sprints.view
+├── dbt_cloud
+│   ├── cloud_accounts.view
+│   ├── cloud_ide_sessions.view
+│   └── cloud_projects.view
+├── dbt_core
+│   ├── dbt_doc_viewer_activity.view
+│   └── dbt_projects.view
+├── finance
+│   ├── quickbook_invoices.view
+│   └── subscription_transactions.view
+└── fishtown.model
+```
+
+Explores should be optimized for the business stakeholder where we're getting the best represented business unit to run our analyses off of. This means that explores should be organized based on department or category of the business unit. For example, we have several Slack views, but maybe they all join under a singular `Slack Messages` explore, which should fall under the `Community` category.
+
+Building off of our above analogy, explores are the packaged items that can combine ingredients (views) to suit a customer's needs. We can utilize various views to create different explores.
+
+
+## Views
+
+#### Structure
+* A view's name should represent the business unit (i.e. the level of granularity of the table). For example, if we're taking `fct_intercom_conversations`, then the view name should be: `intercom_conversations`.
+* The `sql_table_name` should always have the [user attribute feature](https://blog.getdbt.com/how-to-integrate-dbt-and-looker-with-user-attributes/).
```

Comment:
@ericalouie - as we have more granular user permissions, who from the fishtown team has the ability to change this user attribute? is it all developers? I think we should list that here

Reply:
I added some context on where to set your dbt schema attribute!